spec/features/admin/merchants/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe "/admin/merchants/:id, admin::merchants show page" do
+  let!(:merchant_1) { create(:merchant) }
+  let!(:merchant_2) { create(:merchant) }
+
+  describe "as an admin on admin::merchants index page" do
+    it "each merchant link navigates to admin::merchants show page" do
+      visit "/admin/merchants"
+      click_link("#{merchant_1.name}")
+
+      expect(current_path).to eq("/admin/merchants/#{merchant_1.id}")
```

Comment:
Maybe when we refactor we use the "admin invoice path" or "admin merchant path". I can't really remember how that works though haha. I have it this way also